src/samples/Uno.Themes.Samples/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj

```diff
@@ -181,11 +181,23 @@
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
   -->
+
+  <!-- This is a workaround for https://github.com/microsoft/microsoft-ui-xaml/issues/3554 -->
+  <Target Name="WorkaroundForIssue3554_1" BeforeTargets="_CopyOutOfDateSourceItemsToOutputDirectory" AfterTargets="GetCopyToOutputDirectoryItems">
+    <ItemGroup>
+      <_SourceItemsToCopyToOutputDirectory Remove="@(_SourceItemsToCopyToOutputDirectory)" Condition="'%(_SourceItemsToCopyToOutputDirectory.TargetPath)'=='Microsoft.UI.Xaml\Assets\NoiseAsset_256X256_PNG.png'" />
+    </ItemGroup>
+  </Target>
+  <Target Name="WorkaroundForIssue3554_2" BeforeTargets="_GenerateAppxPackageRecipeFile">
+    <ItemGroup>
+      <AppxPackagePayload Remove="@(AppxPackagePayload)" Condition="'%(AppxPackagePayload.TargetPath)'=='Microsoft.UI.Xaml\Assets\NoiseAsset_256X256_PNG.png'" />
+    </ItemGroup>
+  </Target>
```

Comment:
@jeromelaban 
for context, Uno.Material & Uno.Cupertino is on Microsoft.UI.Xaml@2.6.0-prerelease.210113001
and the consuming app have a higher version of this package.

from my limited testing, ive noticed that this is only an issue for the sample app.
and, a separate sample app on Microsoft.UI.Xaml@2.6.2 + Uno.Material@2.3.0 doesnt have this issue.
im suspecting this _only_ occurs to force package update when using ProjectReference, as opposed to PackageReference.
in your opinion, is this okay?

Reply:
That's fine if it's the sample app only. and most devs will likely update to a later version of the package anyways.